unit-tests/test-phase.js

```diff
@@ -224,4 +223,19 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+    it('error thrown', () => {
+      let errorFunc = () => {
+        phase.timeoutError();
+      };
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
Awesome. Keep the `.bind`s, and for those that have to have a variable declaration, just keep a gap between the declarations and the functionality.